biopandas/mmcif/mmcif_parser.py

```diff
@@ -0,0 +1,444 @@
+# The CIF parser relieves heavily on the parser from PDBJapan (author: Gert-Jan Bekker).
+# Python CIF parser: https://gitlab.com/pdbjapan/tools/cif-parsers
+# License: MIT
+#   See https://gitlab.com/pdbjapan/tools/cif-parsers/blob/master/LICENSE
+import gzip
+import re
+
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+try:
+    str.partition
+
+    def partitionString(string, sep):
+        return string.partition(sep)
+
+
+except:
```

Comment:
Do we know the type(s) of exception that would normally occur here?

Reply:
I expect it would be an `AttributeError`. This was added in Python [2.5 ](https://docs.python.org/2/library/stdtypes.html#str.partition) - probably safe to assume we don't need to support?

Reply:
Yeah, we can remove the except then.